### DISCUSSION TITLE
how should I deal with that the loss is well   but prediction is always

### DISCUSSION THREAD

**Discussion by hualihus** [lululxvi/deepxde#1469] [Q&A]
dear lulu 
I had counter this problem，the loss is very low，but the prediction is always 0
here is my code
def pde(x, y):
    y0 = y[:,0:1]
    y1 = y[:,1:2]
    dy0_xx = dde.grad.hessian(y, x,component=0)
    dy1_xx = dde.grad.hessian(y, x,component=1)

    return [-dy0_xx-k0**2*y0,
            -dy1_xx-k0**2*y1]

def pde(x, y):
    y0 = y[:,0:1]
    y1 = y[:,1:2]
    dy0_xx = dde.grad.hessian(y, x,component=0)
    dy1_xx = dde.grad.hessian(y, x,component=1)
    
    return [-dy0_xx-k0**2*y0,
            -dy1_xx-k0**2*y1]

def boundary(x, on_boundary):
    return on_boundary

def boundary_r(x, on_boundary):
    return on_boundary and dde.utils.isclose(x[0], 0)

def boundary_l(x, on_boundary):
    return on_boundary and dde.utils.isclose(x[0], 2.0)

def func_rigth_u0(x, y):
    result = - k0 * y[:, 1:2]
    return result

def boundary(x, on_boundary):
    return on_boundary

def boundary_r(x, on_boundary):
    return on_boundary and dde.utils.isclose(x[0], 0)

def boundary_l(x, on_boundary):
    return on_boundary and dde.utils.isclose(x[0], 2.0)

def func_rigth_u0(x, y):
    result = - k0 * y[:, 1:2]
    return result
def create_model(config):
    learning_rate, num_dense_layers, num_dense_nodes, activation, num_sigma = config
    geom = dde.geometry.Interval(0, 2.0)
    bc0_outer = dde.RobinBC(geom, func_rigth_u0, boundary_r, component=0)
    bc1_outer = dde.RobinBC(geom, func0_rigth_u1, boundary_r, component=1)
    bc0_inner = dde.NeumannBC(geom, func0_left_u0, boundary_l, component=0)
    bc1_inner = dde.NeumannBC(geom, func0_left_u1, boundary_l, component=1)
    bcs = [ bc0_inner,bc1_inner,bc0_outer,bc1_outer]
    data = dde.data.PDE(
    geom,
    pde,
    bcs,
    1600,
    2,
    train_distribution="pseudo",
    solution=func,
    num_test=1000,
    #auxiliary_var_function=ex_func
    )

    layer_size = [1] + [num_dense_nodes] * num_dense_layers + [2]
    activation = activation
    initializer = "Glorot uniform"
    net = dde.nn.MsFFN(layer_size, activation, initializer, sigmas=[1, num_sigma])
    
    model = dde.Model(data, net)
    model.compile(
    "adam",
    lr=learning_rate,
    metrics=["l2 relative error"],
    decay=("inverse time", 2000, 0.9),
    )
